config/locales/en.yml

```diff
@@ -1,203 +1,204 @@
-          - "No"
-          - "Not sure"
-      afford_rent_mortgage_bills:
-        title: "Are you finding it hard to afford rent, your mortgage or bills?"
-        options:
-          - "Yes"
-          - "No"
-          - "Not sure"
-      afford_food:
-        title: "Are you finding it hard to afford food?"
-        options:
-          - "Yes"
-          - "No"
-          - "Not sure"
-      get_food:
-        title: "Are you able to get food?"
-        title_caption: "Getting food"
-        options:
-          - "Yes"
-          - "No"
-          - "Not sure"
-      have_you_been_made_unemployed:
-        title: "Have you been made unemployed or put on temporary leave (furloughed)?"
-        title_caption: "Being unemployed or not having any work"
-        options:
-          - "Yes"
-          - "I might be soon"
-          - "No"
-          - "Not sure"
-      are_you_off_work_ill:
-        title: "Are you off work because you’re ill or self-isolating?"
-        title_caption: "Going in to work"
-        options:
-          - "Yes"
-          - "No"
-        custom_select_error: "Select yes if you’re off work because you’re ill or self-isolating"
-      self_employed:
-        title: "Are you self-employed or a sole trader?"
-        title_caption: "Being unemployed or not having any work"
-        options:
-          - "Yes"
-          - "No"
-          - "Not sure"
-      still_working:
-        title: "Are you still going in to work even though you’re not a key worker?"
-        options:
-          - "Yes"
-          - "I do not know if I’m a key worker"
-          - "No"
-      living_with_vulnerable:
-        title: "Are you worried about going into work because you live with someone vulnerable to coronavirus?"
-        options:
-          - "Yes"
-          - "No"
-          - "Not sure"
-      have_somewhere_to_live:
-        title: "Do you have somewhere to live?"
-        options:
-          - "Yes"
-          - "I do now but I might lose it"
-          - "No"
-          - "Not sure"
-      have_you_been_evicted:
-        title: "Have you been evicted?"
-        options:
-          - "Yes"
-          - "I might be evicted soon"
-          - "No"
-          - "Not sure"
-      mental_health_worries:
-        title: "Are you worried about your mental health or someone else‘s mental health?"
-        options:
-          - "Yes"
-          - "No"
-          - "Not sure"
+        <p class="govuk-body">We do this for your security. We've deleted all the details you entered to protect your data.</p>
+    get_help_from_nhs:
+      title: Get urgent help from the NHS now
+      link_text: Go to NHS 111 online
+      link_href: "https://111.nhs.uk/"
+    coronavirus_form:
+      errors:
+        page_title_prefix: 'Error: '
+        heading: "There is a problem"
+        radio_field: "Select %{field}"
+        checkbox_field: "Select at least one %{field}"
+      questions:
```

Comment:
From someone completely new to this app and it's structure, I'm not sure why `questions` here is in `en.yml` and not in `questions/en.yml`.

Reply:
That's a really good point, should be moved
I think we thought we'd start out with a blob and then break it out as needed.
... i think it's needed :D